docs/configuration.md

```diff
@@ -102,7 +103,7 @@ transformers: [
 ]
 ```
 
-Some plugins are included by default in the[ `quartz.config.ts`](https://github.com/jackyzha0/quartz/blob/v4/quartz.config.ts), but there are more available.
+Some plugins are included by default in the[`quartz.config.ts`](https://github.com/jackyzha0/quartz/blob/v4/quartz.config.ts), but there are more available.
```

Comment:
can you uncommit this whitespace change?

Reply:
My formatted corrected the white space in the link. I added the space but in front of the link instead of in the link. Is that ok or would you rather I changed it back?